### ISSUE TITLE
Exclude year from in-text citations

### ISSUE THREAD

**Issue by lauriepetty** [zotero/zotero#439]
I have recently begun trying to convert my reference management to Zotero. I have come across an issue that seems to have been overlooked.

**TL;DR:**

_Zotero should allow suppression of the year in citations._

**Long Version**

When citing a publication and the author and year have already been mentioned previously in the text, the citation to be managed should only include the page number. The exact display of the page number varies depending on the actual style being used. See below for an example of MLA vs APA:

![Styles](http://nx.is/UXkr+)

Source: https://owl.english.purdue.edu/media/pdf/20110928111055_949.pdf

As you can see, this format varies across styles and should be managed by the reference management software just like all citations.
